Lib/http/client.py

```diff
@@ -578,11 +578,7 @@ def _read_chunked(self, amt=None):
         assert self.chunked != _UNKNOWN
         value = []
         try:
-            while True:
-                chunk_left = self._get_chunk_left()
-                if chunk_left is None:
-                    break
-
+            while (chunk_left := self._get_chunk_left()) is not None:
```

Comment:
Somewhere in this file:

    Fixing Python file whitespace ... 1 file:
      Lib/http/client.py

If you develop on linux you can use `make patchcheck`, otherwise, find and remove any trailing whitespace.

Reply:
On Windows, manual invocation of `python Tools/scripts/patchcheck.py` can be used.

Reply:
No. Perhaps it was already there and no one's touched the module in a while?  Maybe your editor wanted space there?  Maybe a butterfly flapped its wings in the Amazon?  :-)

Reply:
I think the reason is that `make` runs patchcheck on files that are changed, so this PR is the one that caught trailing whitespace added in the past and not caught.